src/Pages/jinheekim/Login/Login.js

```diff
@@ -35,17 +30,27 @@ class Loginjinhee extends React.Component {
                   <img alt="로고" src="/images/Login/logo_text.png" />
                 </div>
                 <div className="loginForm">
-                  <input onChange={this.handleIdInput} className="email" type="text" placeholder="전화번호, 사용자 이름 또는 이메일" />
-                  <input onChange={this.handlePwInput} className="pwd" type="password" placeholder="비밀번호" />
-                  <button style={{ backgroundColor: this.state.idValue.includes("@") && this.state.pwValue.length >= 5 ? "#0095F6" : "#B9E0FC" }}
-                  className="loginBtn" onClick={this.goToMain}>로그인</button>
-                  {/* 
+                  <input
+                    className="emailInput" 
+                    type="text" 
+                    name="idValue"
+                    onChange={this.handleInput}                  
+                    placeholder="전화번호, 사용자 이름 또는 이메일" 
+                  />
+                  <input 
+                    className="pwdInput" 
+                    type="password"
+                    name="pwValue"
+                    onChange={this.handleInput} 
+                    placeholder="비밀번호" 
+                  />
+                  
                   <button
-                    onClick={this.handleIdInput} className={this.state.loginBtn ? "activated" : "deactivated"}
+                    onClick={this.goToMain} className={this.state.idValue.includes("@") && this.state.pwValue.length >= 5? "activated loginBtn" : "deactivated loginBtn"}
```

Comment:
state 객체 비구조화 할당만 적용해줘도 조금 더 깔끔해질 수 있겠네요!
```suggestion
const { idValue, pwValue } = this.state;
                    onClick={this.goToMain} className={idValue.includes("@") && pwValue.length >= 5? "activated loginBtn" : "deactivated loginBtn"}
```

Reply:
반영했지만 컴파일 실패가 떴습니다 😭